identity/mock-dps-server/src/request.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use crate::server::Response;
+
+trait DpsRequest {
```

Comment:
You always call `.process()` on the concrete type, so it doesn't seem necessary to have a trait.

Even the three concrete types can just be three functions.

Reply:
Fixed